### DISCUSSION TITLE
Achievements View - Cannot be seen from mobile app

### DISCUSSION THREAD

**Discussion by tahsintunan** [community/community#18169] [Mobile]
I think it'd be very nice if the achievements could be seen from the mobile GitHub app as well. For now, it seems only web versions have this feature.

**Comment by NhanAZ** [comment]
Most mobile browsers have a "Sites for desktop" feature so you might want to consider using that.
I think displaying unimportant things like badges takes up space on the phone and this also takes a long time to change.
